frontend/src/components/organisms/forms/sponsorshipForm.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { Form, Input, Button, InputNumber, Upload} from "antd";
+import { UploadOutlined } from '@ant-design/icons';
+import { ModalDrawer, SponsorshipForm } from "components/organisms";
+import { SponsorshipFormValues } from "types";
+
+const Component = (props: { event?:any }) => {
+  const {
+    control,
+    getValues,
+    formState: { errors },
+  } = useForm<SponsorshipFormValues>({ mode: "onChange" });
+
+  const [sent, setSent] = React.useState(false);
+  const normFile = (e: any) => {
+    console.log('Upload event:', e);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-Eventia/eventus/pullRequest?prid=9091638)
```suggestion
    console.log("Upload event:", e);
```